screenpy_selenium/questions/element.py

```diff
@@ -23,6 +23,8 @@ class Element:
         the_actor.should(See.the(Element(WELCOME_BANNER), IsVisible()))
     """
 
+    caught_exception: Optional[TargetingError]
```

Comment:
We were talking about this on Discord, and i do really like how we can be extremely explicit here about what kind of exceptions we're expecting to handle in the actual Questions.